src/builder.rs

```diff
@@ -302,6 +302,23 @@ impl Builder {
         Ok(())
     }
 
+    /// The net value of the bundle to be built. The value of all spends,
+    /// minus the value of all outputs.
+    ///
+    /// Useful for balancing a transaction, as the value balance of an individual bundle
+    /// can be non-zero, but a transaction may not have a positive total value balance.  
+    pub fn value_balance(&self) -> i64 {
+        self.spends
+            .iter()
+            .map(|spend| spend.note.value().inner() as i64)
```

Comment:
I would prefer that this return a `ValueSum` value, which wraps an i128 internally.